client-dotnet/src/Controller/SutController.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Controller.Api;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace Controller {
+    public abstract class SutController {
```

Comment:
add some comments to this class, also mark abstract methods which have not included yet.

Reply:
Ok Man, Thanks for your feedback